tests/lib/prepare.sh

```diff
@@ -444,6 +444,10 @@ repack_snapd_snap_with_deb_content_and_run_mode_firstboot_tweaks() {
     local UNPACK_DIR="/tmp/snapd-unpack"
     unsquashfs -no-progress -d "$UNPACK_DIR" snapd_*.snap
 
+    # data/preseed.json is not included in the deb, use the latest
+    # version from source tree to replace the one in the re-packed snapd snap.
+    cp /home/gopath/src/github.com/snapcore/snapd/data/preseed.json "$UNPACK_DIR"/usr/lib/snapd
```

Comment:
I think this should work;
```suggestion
    cp "$PROJECT_PATH/data/preseed.json" "$UNPACK_DIR"/usr/lib/snapd
```